### PR TITLE
Feature/focus button styles

### PR DESCRIPTION
**Pull request by bstream**

### CODE REVIEW

File: src/components/button/button.md [hunk 1]
```diff
@@ -31,46 +31,13 @@ Secondary button:
     const style = {
       marginRight: 16,
     };
-    const groupstyle = {
-      marginTop: 16,
-    };
 
     <div>
-      <div style={groupstyle}>
```
Comment (willeeklund) [inline src/components/button/button.md]: =(

File: src/components/button/button.md [hunk 2]
```diff
@@ -108,9 +75,21 @@ Sizes:
...
+      marginTop: 16,
+    };
+
     <div>
-      <Button variant="primary" size="xs" style= { style }>Button xs</Button>
-      <Button variant="primary" size="sm" style= { style }>Button sm (Default)</Button>
-      <Button variant="primary" size="md" style= { style }>Button md</Button>
-      <Button variant="primary" size="lg" style= { style }>Button lg</Button>
+      <div style={groupstyle}>
+        <Button variant="primary" size="xs" style= { style }>Button xs</Button>
```
Comment (willeeklund) [inline src/components/button/button.md]: 👍

File: src/styles/color.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ export default {
   blueDarker: '#2B417A', // Used together with dark in header gradient
   green: '#26A699', // Plus performance, success
   red: '#D54E16', // Minus performance, error
+  redLight: '#DA6E42', // Sell
```
Comment (willeeklund) [inline src/styles/color.js]: I see.

**Comment by gish** [comment]
Seems like the focus effect doesn't work in Firefox nor Safari.

**Pull request merged**